examples/realtime.ts

```diff
@@ -39,14 +39,25 @@ const realtimeClient = await client.realtime.connect(stream, {
 } satisfies RealTimeClientConnectOptions);
 
 // 3. Prompt Management
-// 3.1 Sending a prompt, the prompt will be enhanced automatically (great for out-of-the-box experience)
-realtimeClient.setPrompt("Lego World");
+// 3.1 Sending a prompt (fire-and-forget, don't wait for acknowledgment)
+realtimeClient.setPrompt("Lego World"); // Returns a promise, but we don't await it
 
-// 3.2 Sending an already enhanced prompt (great for advanced use-cases)
+// 3.2 Sending a prompt and waiting for server acknowledgment
+try {
+	const success = await realtimeClient.setPrompt("Lego World", {
+		enhance: true, // optional, defaults to true
+		maxTimeout: 15000, // optional, defaults to 15000ms
+	});
+	console.log("Prompt acknowledged by server:", success);
+} catch (error) {
+	console.error("Prompt failed or timed out:", error);
+}
```

Comment:
1. Maybe put it below the enhancement example? so it'll be: 3.1 - setPrompt, 3.2 - setPrompt with enhance, 3.3 - setPrompt with waiting for response. I think that'd be more clear.
2. wdyt about simplifying? this is example no we don't really need all the error handling. a one-liner should do the trick imo and is now focus on the actual thing we want to show (the `await`)
```suggestion
// 3.2 Sending a prompt and waiting for server acknowledgment
	await realtimeClient.setPrompt("Lego World");
```